odoo/addons/base/models/ir_qweb.py

```diff
@@ -472,6 +475,14 @@ def keep_query(*keep_params, **additional_params):
                 params[param] = request.httprequest.args.getlist(param)
     return werkzeug.urls.url_encode(params)
 
+
+def _qweb_ast_get_attr(obj, key):
+    return key in ("dumps", "get", "count", "strip") or (hasattr(obj, "_fields") and key in obj._fields)
```

Comment:
For `("dumps", "get", "count", "strip")` can you check the type of obj is dict / string ? :)

Reply:
the type is already checked because of  `__ast_check_attr_and_type` :wink: 
I checked the type just in case and it's not only dict or string.

Reply:
yes, but so you can call "get" on all authozized types, not only on dict